### ISSUE TITLE
Pulumi Deployments: Add an indication of deployment in progress in project view in Pulumi UI

### ISSUE THREAD

**Issue by MitchellGerdisch** [pulumi/pulumi-cloud-requests#128]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

Currently if there is an update or preview in progress for a stack, the project overview screen shows a little rotating circle as highlighted in this screen shot:
![image](https://user-images.githubusercontent.com/2699172/200360232-57d1ddac-99b1-425a-b30a-feb4463a9ced.png)

Once a deployment moves into the "pulumi operation" stage we do see the little spinning circle thing, but it would be nice to have a similar indication when a deployment has started and is in progress leading up to the operation stage.

**Comment by susanev** [comment]
yes! we also have more plans for the stacks list and project pages to integrate in the deployment feature.

**Comment by susanev** [comment]
if this is taken on in q4, the ux team has designs that yall can use as input if you want.

**Comment by tatcoo-pulumi** [comment]
OK, given this is done marking as complete, @beneficialmedia assuming you've got tracking in place on the PDS refresh piece
